composer.json

```diff
@@ -21,9 +21,9 @@
     "nyholm/psr7": "^1.2",
     "phpbench/phpbench": "1.0.0-beta1",
     "phpstan/extension-installer": "^1.0",
-    "phpstan/phpstan": "0.12.82",
-    "phpstan/phpstan-phpunit": "0.12.18",
-    "phpstan/phpstan-strict-rules": "0.12.9",
+    "phpstan/phpstan": "^0.12",
```

Comment:
This will break builds. Every phpstan 0.x.y release is like a new major. There are new checks/fixes introduced that might not be covered in our code yet.

Reply:
Good catch, thank you. Pinned the version again 👍